### PR TITLE
Update setup.py with progress module for pi

### PR DESCRIPTION
**Pull request by cfox570**
Added progress module to be installed.  Found it missing when running donkey createcar.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def package_files(directory, strip_leading):
               'Adafruit_SSD1306',
               'RPi.GPIO',
               'pyserial',
+              'progress',
```
Comment (tikurahul) [inline setup.py:57]: You should add it to `install_requrires` and not specific to `pi`.

Comment (DocGarbanzo) [reply setup.py:57]: Yes, and take it out of 'pc'.

Comment (cfox570) [reply setup.py:57]: And take it out of 'dev'?    If I update the code, how do I use Github to update the pull request?

Comment (cfox570) [reply setup.py]: Made the changes online

Comment (DocGarbanzo) [reply setup.py]: Thanks @cfox570, this looks right. If you update your branch (i.e. push) this updates the PR automatically.

Comment (cfox570) [reply setup.py]: I did the reverse. I updated through the PR and it updated my fork.  Thanks

**Review by tikurahul — APPROVED** [review]

**Pull request merged**